### PR TITLE
greggie's first commit

### PR DESCRIPTION
**Pull request by gbrock92**
put me in main branch, gurl!

### CODE REVIEW

**Comment by gbrock92** [comment]
put me in, coach. im ready to play today.

File: greg/jokes.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  
+  .image {
+    -webkit-transition: all 1s ease-in-out;
+  -moz-transition: all 1s ease-in-out;
+  -o-transition: all 1s ease-in-out;
+  -ms-transition: all 1s ease-in-out;
+  }
+  
+  .image:hover {
+    -webkit-transform: rotate(360deg);
```
Comment (mikelbrierly) [inline greg/jokes.css:40]: Very cool, what does this do per chance Greggie?

File: greg/jokes.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      <div id="punchline">
+        <p>A barn rasin'!</p>
+      </div>
+      <div class="image">
+      <a href="https://www.nhl.com/sharks" target="_blank" >
+        <img src="https://ofhl.files.wordpress.com/2016/03/burns-960x540.jpg" width=450px  alt="star defenseman Brent Burns of the San Jose Sharks laughing">
+      </a>
+      </div>
+      <div>
+      <a href="https://www.w3schools.com/cssref/sel_hover.asp" target="_blank"> Where I learned the "hover" effect</a>
```
Comment (mikelbrierly) [inline greg/jokes.html:22]: 👍

File: greg/jokes.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  <body>
+    <h1 class="setup">What's an Amish's favorite kind of rasin?</h1>
+    <main>
+      <div id="punchline">
+        <p>A barn rasin'!</p>
+      </div>
+      <div class="image">
+      <a href="https://www.nhl.com/sharks" target="_blank" >
+        <img src="https://ofhl.files.wordpress.com/2016/03/burns-960x540.jpg" width=450px  alt="star defenseman Brent Burns of the San Jose Sharks laughing">
```
Comment (mikelbrierly) [inline greg/jokes.html:18]: Awesome alt text!!

File: greg/jokes.css (hunk 1 above)
Comment (gbrock92) [reply greg/jokes.css:40]: that shat makes da pic spin when ya hover

File: greg/jokes.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    height: 150px;
+    width: 500px;
+  }
+  
+  p {
+    text-decoration: none;
+      color: transparent;
+  }
+  
+  p:hover {
```
Comment (mikelbrierly) [inline greg/jokes.css:26]: Nice pseudo-selector!

File: index.html [hunk 5]
```diff
@@ -16,7 +16,7 @@ <h1>Dev Delta Jokes</h1>
         <li><a href="collin/index.html">Collin's Joke</a></li>
         <li>Travis Placeholder</li>
         <li>Eric Placeholder</li>
-        <li>Greg Placeholder</li>
```
Comment (gbrock92) [inline index.html:19]: Placeholder, real good story so far reference.

**Review by mikelbrierly — APPROVED** [review]
Good Job!

**Pull request merged**